packages/vuetify/src/components/VItemGroup/VItemGroup.sass

```diff
@@ -4,4 +4,4 @@
   flex: 0 1 auto
   position: relative
   max-width: 100%
-  transition: $item-group-transition
+  transition: 0.28s $standard-easing
```

Comment:
what's the reason for not using $item-group-transition? that way it's less customizable